### PR TITLE
Update MAnorm recipe

### PR DESCRIPTION
**Pull request by haydensun**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by zfrenchee** [comment]
@bgruening This is the MAnorm author's PR. Once we merge this, we'll  move on to the galaxy tool.

File: recipes/manormfast/build.sh [hunk 1]
```diff
@@ -1,3 +0,0 @@
-#!/bin/bash
-
-$PYTHON setup.py install
```
Comment (bgruening) [inline recipes/manormfast/build.sh]: Is this tool deprecated? Should we keep the recipe?

Comment (bgruening) [reply recipes/manormfast/build.sh]: fair enough, will still keep it in the channel for rep. reasons.

Comment (zfrenchee) [reply recipes/manormfast/build.sh]: Thanks!

Comment (haydensun) [reply recipes/manormfast/build.sh]: As explained by @zfrenchee, the newer version just fixes everything up and changes the misleading name to a proper one. Sorry about that!

**Pull request merged**